src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -41,6 +41,7 @@
       PreBuildEvent;
       ResolveReferences;
       GetTargetPath;
+      GetIncludedFiles;
```

Comment:
Is this necessary? We're already listing this target as a dependency for `CoreCompile`?

Reply:
No idea, to be honest, I'll remove it and if it still works I'll leave it removed 😄

Reply:
Removing seems to be fine, change pushed.